packages/grafana-ui/src/components/PanelChrome/PanelContext.ts

```diff
@@ -22,6 +22,20 @@ export interface PanelContext {
   onAnnotationCreate?: (annotation: AnnotationEventUIModel) => void;
   onAnnotationUpdate?: (annotation: AnnotationEventUIModel) => void;
   onAnnotationDelete?: (id: string) => void;
+
+  /**
+   * Enables modifying thresholds directly from the panel
+   *
+   * @alpha -- experimental
+   */
+  canEditThresholds?: boolean;
```

Comment:
:thinking: is this a can/should question?  IE permissions or the panel supports it?  

I suspect this makes more sense as a panel/option/prop then panel context

(unless we don't easily expose if the user `canEdit` the panel)

Reply:
@ryantxu we decided with @torkelo not to support threshold handles in the panel itself for now, only in the alert rule view. 

This particular line tho is similar to canAddAnnotations. The user permissions are part of the dashboard model which is not directly passed to individual panels. I want to follow the same structure for both annotations and thresholds edit, hence the bool flag here. Think this could be a function instead so that we could evaluate other factors, but there's no need for that ATM. This simplifies the config for Alert rule editor.